src/SharedTaskCode/UserInputToScheduledTaskMapper.psm1

```diff
@@ -207,11 +207,13 @@ function Convert-MinutesToTimeSpan([string] $minutes)
 	return $minutesAsTimeSpan
 }
 
-function Get-ScheduledTaskSettings([bool] $shouldBeEnabled)
+function Get-ScheduledTaskSettings([bool] $shouldBeEnabled, [bool] $startWhenAvailable)
```

Comment:
Looks like the case change didn't make it into the other "shared" files. You can propagate the changes to them by running the "src\SharedTaskCode\_CopyModuleFilesToTasksSharedCodeDirectories.ps1" script.

Reply:
I think once this change has been replicated to the other files this PR will be ready to be merged in 👍

Reply:
> Looks like the case change didn't make it into the other "shared" files. You can propagate the changes to them automatically by running the "src\SharedTaskCode_CopyModuleFilesToTasksSharedCodeDirectories.ps1" script, rather than doing them all by hand manually.

Big thanks for catching that, I've used the copymoduleFiles script the last time as well! Very handy :) I'll make a new commit to sort that CamelCasing inconsistency in the Shared folders now.

Reply:
Awesome, thanks 👍